style.css

```diff
@@ -0,0 +1,89 @@
+/* style sheet for rocket science home */
```

Comment:
Rule of thumb: if you need to leave a comment, your code needs to be made more expressive.

In this case, I don't think you need to leave a comment :P